internal/podman/client.go

```diff
@@ -360,7 +361,11 @@ func (o *client) RunContainer(ctx context.Context, opts RunContainerOpts) ([]byt
 	}
 
 	for hostPort, containerPort := range opts.Ports {
-		arg = append(arg, "-p", "127.0.0.1:"+strconv.Itoa(hostPort)+":"+strconv.Itoa(containerPort))
+		portMapping := strconv.Itoa(hostPort) + ":" + strconv.Itoa(containerPort)
+		if !opts.BindAllIPs {
+			portMapping = "127.0.0.1:" + portMapping
+		}
+		arg = append(arg, "-p", portMapping)
```

Comment:
[nit] better redability
```suggestion
                if opts.BindAllIPs {
		      portMapping := strconv.Itoa(hostPort) + ":" + strconv.Itoa(containerPort)
		} else {
		      portMapping = "127.0.0.1:" +  strconv.Itoa(containerPort)
		}
		arg = append(arg, "-p", portMapping)
```